src/v/pandaproxy/reply.h

```diff
@@ -76,50 +77,79 @@ errored_body(std::error_condition ec, ss::sstring msg) {
     return rep;
 }
 
+inline std::unique_ptr<ss::httpd::reply>
+errored_body(std::error_condition ec, ss::sstring msg, error_probe& eprobe) {
+    auto rep = errored_body(ec, std::move(msg));
```

Comment:
I think the overload of `errored_body` that's called here, should be inlined, or there's a good chance it can be called and the probe missed off.